static/schema/projects.json

```
@@ -49,6 +49,11 @@
             "additionalProperties": false,
             "unevaluatedProperties": false
         },
+        "issueNumber": {
+            "type": "string",
+            "additionalProperties": false,
+            "unevaluatedProperties": false
```

If going for the issueLinks option:

```suggestion
        "issueNumber": {
            "type": "object",
                "link": {
                    "type": "string",
                    "description": "A link to the issue"
                },
                "title": {
                    "type": "string",
                    "description": "A short title for the issue"
                }
            },
            "additionalProperties": false,
            "unevaluatedProperties": false
```